packages/import-bundle/src/index.js

```diff
@@ -8,13 +10,49 @@ export async function importBundle(bundle, options = {}) {
   const {
     filePrefix,
     endowments: optEndowments = {},
+    globalLexicals = {},
+    // transforms are indeed __shimTransforms__, intended to apply to both
+    // evaluated programs and modules shimmed to programs.
+    transforms = [],
     inescapableTransforms = [],
     inescapableGlobalLexicals = {},
-    ...compartmentOptions
   } = options;
-  const endowments = { ...optEndowments };
-  const { source, sourceMap, moduleFormat } = bundle;
+  const endowments = {
+    TextEncoder,
```

Comment:
@michaelfig will exposing these enable a metering break? If they're wrapped like all the other globals, probably not, but it's a new ability being handed to all Compartments so I wanted to double-check.

Reply:
@kriskowal I'm guessing that `TextEncoder` and `TextDecoder` will be available in the immediate `Compartments` produced by import-bundle, but they won't automatically be in any child compartments created therein. Is that correct? I guess that won't prohibit nested import-bundles, like the kernel does to load vats.

Reply:
`encode` and `decode` are O(string.length), so we may need to meter them.

Reply:
cc @michaelfig I think I need your reaction to the question “do we need metering to attenuate Text{En,De}coder?”, before I land this change.

Reply:
> cc @michaelfig I think I need your reaction to the question “do we need metering to attenuate Text{En,De}coder?”, before I land this change.

Ideally, the TextEncoder and TextDecoder sources would be evaluated in a metered compartment (not endowed as a vetted shim), so that the metering transform would be applied to them as they are to user code.

Reply:
TextEncoder and TextDecoder are platform code.

*sigh*… We *could* get metering for free by slurping our own UTF-8, but I’d wanted to avoid that because it’ll be unnecessary bloat on the web. The Text* utilities are available in both Node.js and Web.

Reply:
> TextEncoder and TextDecoder are platform code.

Oh, in that case, they're covered by the blanket metering performed by `@agoric/install-metering-and-ses`.  No need to special-case (I had thought they were shims written by you).